tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -489,6 +489,8 @@ def do_rolling_bounce(self, processor, counter, current_generation):
         first_other_node = first_other_processor.node
         second_other_node = second_other_processor.node
 
+        kafka_version_str = self.get_version_string(self.base_version_number)
```

Comment:
This will work, but make the check slightly weaker, since it will never check for `SNAPSHOT`, even if I am currently on a snapshot version.
```suggestion
        kafka_version_str = self.get_version_string(str(DEV_VERSION))
```